### ISSUE TITLE
[Discussion] 4.x Release

### ISSUE THREAD

**Issue by mhassan1** [Financial-Times/polyfill-library#1144]
It's been a few years since the last major release. The next major release could address these:
- [x] Drop support for Node 8 and Node 10 (https://github.com/Financial-Times/polyfill-library/pull/1148)
- [x] Update `baselineVersions` in `polyfill-useragent-normalizer` (https://github.com/Financial-Times/polyfill-useragent-normaliser/pull/173)
- [x] Update browser lists to remove outdated browsers (https://github.com/Financial-Times/polyfill-library/pull/1150)
- [x] Remove no longer needed polyfills and dependencies for IE 6 and 7 (e.g. `Window`, `JSON`, `localStorage`) (https://github.com/Financial-Times/polyfill-library/pull/1149)
- [x] Remove no longer needed polyfills and dependencies for IE 8 (e.g. `Array.isArray`, `Date.now`, `document`) (https://github.com/Financial-Times/polyfill-library/pull/1153)
- [x] Clean up polyfill hacks for IE 8 (e.g. `supportsGetters`) (https://github.com/Financial-Times/polyfill-library/pull/1151, https://github.com/Financial-Times/polyfill-library/pull/1155)
- [x] Update `dot-notation` ESLint rule to allow `proclaim.throws`, `obj.delete`, etc. (https://github.com/Financial-Times/polyfill-library/pull/1154)
- [x] Bump `@formatjs/intl-localematcher` to get this fix: https://github.com/formatjs/formatjs/issues/3058 (https://github.com/Financial-Times/polyfill-library/pull/1148)

**Comment by mhassan1** [comment]
@JakeChampion What do you think?

**Comment by JakeChampion** [comment]
I think this is excellent

**Comment by mhassan1** [comment]
Can we create a `4.x` branch so we can start putting up PRs?

**Comment by JakeChampion** [comment]
@mhassan1 I've made a branch named 4.x 👍 https://github.com/Financial-Times/polyfill-library/tree/4.x

**Comment by mhassan1** [comment]
@JakeChampion I am just waiting on a major release of `polyfill-useragent-normalizer` before continuing.

**Comment by JakeChampion** [comment]
@mhassan1 thanks, the new major of polyfill-useragent-normaliser should now be released

**Comment by JakeChampion** [comment]
Thank you @mhassan1 for all this work, we have now released v4 🎉  -- https://github.com/Financial-Times/polyfill-library/releases/tag/v4.0.0
